### ISSUE TITLE
No events sent after console.log with umlauts (Android)

### ISSUE THREAD

**Issue by hermitdemschoenenleben** [getsentry/sentry-capacitor#100]
### Environment

I'm developing a capacitor app with @sentry/angular 6.14.0 and @sentry/capacitor 0.4.0 and sentry.io. I noticed that event reporting stopped after a short time in app. Analyzing the reason for this, I found out that `console.log` calls with umlauts cause the problem.
  
### Steps to Reproduce

```
Sentry.captureException('first test message');
console.log('just normal stuff');
Sentry.captureException('still works');
console.log('üö');
Sentry.captureException('doesnt work anymore');
```

### Expected Result

All messages should appear on sentry.io.

### Actual Result

However, only the first two appear. This only happens when running the app on android. In the browser, everything works as expected.

**Comment by bruno-garcia** [comment]
Thanks for raising this. Indeed this looks like a bug.
Could you please enable `debug` mode so we can get more info on what's breaking?

**Comment by jennmueng** [comment]
@hermitdemschoenenleben Yep, debug mode like Bruno said, and if possible both logs from Javascript and Android logcat logs from Sentry.

**Comment by hermitdemschoenenleben** [comment]
I attached the logcat (notice that I used Sentry.captureMessage, not Sentry.captureException like I wrote above)
[logcat.txt](https://github.com/getsentry/sentry-capacitor/files/7651010/logcat.txt)

**Comment by hermitdemschoenenleben** [comment]
in javascript console, there was no output

**Comment by KevinKelchen** [comment]
I think I reported this same issue (#124) and it's just been fixed and released in v0.4.3.

**Comment by hermitdemschoenenleben** [comment]
great, thanks!
